docs/installation/local.mdx

```diff
@@ -4,15 +4,42 @@ sidebar_position: 1
 
 # Local Installation
 
-Use Capact CLI to create local Kubernetes cluster and install Capact on it.
+Use Capact CLI to create a local Kubernetes cluster and install Capact on it.
 
 ## Prerequisites
+
 - [Docker](https://www.docker.com/) installed
 - Capact CLI installed according to the [Getting Started](./../cli/getting-started.mdx#install) guide
 
+### CPU and memory requirements
+
+Capact consists of multiple components, which you can find in [`deploy/kubernetes/charts`](https://github.com/capactio/capact/tree/main/deploy/kubernetes/charts).
+
+One of the components is Neo4j. Unfortunately, its usage, significantly increases CPU and memory consumption. Due to this fact, you need to make sure that Docker has enough resource to install Capact properly.
+
+Our recommendation is:
+- Minimal:
+  - 2CPUs
+  - 4GB
+- Recommended:
+  - 4CPUs
+  - 8GB
+
+### Docker configuration
+
+Please check the [Docker manual](https://docs.docker.com/desktop/mac/#resources) in order to learn how to configure CPU and memory.
+
+<div style={{textAlign: 'center'}}>
+
+![docker-settings](./assets/docker-settings.png)
+
+</div>
+
+
+
```

Comment:
We need to clarify that this is macOS-only thing. Also, I wouldn't create a dedicated section, as it refers to CPU and memory requirements.
What do you think about creating a note from it?

```suggestion
:::note 

While on Linux Docker runs natively, being able to use all CPU and memory if needed, on macOS you need to configure resources manually for Docker virtual machine.

Please check the [Docker manual](https://docs.docker.com/desktop/mac/#resources) in order to learn how to configure CPU and memory.

<div style={{textAlign: 'center'}}>

![docker-settings](./assets/docker-settings.png)

</div>

:::
```

Result:
<img width="1034" alt="Screenshot 2022-01-26 at 09 53 46" src="https://user-images.githubusercontent.com/7155799/151132380-d7641990-84ea-4ddf-a110-0b8d2f00b160.png">

If you don't like it, then we can keep it as a separate subheader (h4) under "CPU and memory requirements"), but it would need to be named as something like "macOS-specific Docker configuration".